worker/src/main.rs

```diff
@@ -99,6 +100,9 @@ fn main() {
     // Setup logging
     env_logger::init();
 
+    // initialize o-call bridge
+    OCallBridge::initialize(Arc::new(OCallBridgeComponentFactoryImpl {}));
+
```

Comment:
Here we initialize the component (i.e. global state) and inject the concrete implementation